### PR TITLE
[RW-7697] Chavanr/user admin UI update

### PR DESCRIPTION
**Pull request by chavanr2019**
updated all the occurrences of "free credits" to "initial credits" 

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: e2e/app/page/admin-user-profile-info.ts [hunk 1]
```diff
@@ -133,24 +133,25 @@ export default class UserProfileInfo extends AuthenticatedPage {
...
   }
 
-  // get the free credits used value
-  async getFreeCreditMaxValue(): Promise<string> {
-    const freeCreditsMax = this.getFreeCreditsUsedInput();
-    const freeCreditsMaxValue = await freeCreditsMax.getProperty<string>('value');
-    const regex = new RegExp(/(?<=of ).*(?= limit)/);
-    return regex.exec(freeCreditsMaxValue)[0];
+  // get the limit value from the initial credits used field
+  async getInitialCreditUsedValue(): Promise<string> {
```
Comment (jmthibault79) [inline e2e/app/page/admin-user-profile-info.ts:141]: This doesn't get the amount used - it gets the limit

File: e2e/app/page/admin-user-profile-info.ts [hunk 2]
```diff
@@ -144,7 +144,7 @@ export default class UserProfileInfo extends AuthenticatedPage {
     const regexpSize = /\$([0-9.]+) used of \$([0-9.]+) limit/;
     const match = initialCreditsUsedValue.match(regexpSize);
     // extract the credit limit value
-    const creditLimitValue = "$" + `${match[2]}`;
+    const creditLimitValue = '$' + `${match[2]}`;
```
Comment (chavanr2019) [inline e2e/app/page/admin-user-profile-info.ts:147]: '$' + `${match[2]}` had to add back the $ sign to match it with the initial-credit-limit value in [line 174](https://github.com/all-of-us/workbench/pull/6139/commits/c7e20705a5933bb0c1af474259751d59dce0a6ee#diff-589375b4fc33f41fa9897ce10f70e5c59b25225af4ec04c1cc93b0f7d8f9734bR174).

File: e2e/app/page/admin-user-profile-info.ts [hunk 3]
```diff
@@ -153,7 +153,7 @@ export default class UserProfileInfo extends AuthenticatedPage {
     const initialCreditsDropdown = SelectMenu.findByName(this.page, { dataTestId: DataTestIdAlias.InitialCreditLimit });
     return await initialCreditsDropdown.getSelectedValue();
```
Comment (chavanr2019) [inline e2e/app/page/admin-user-profile-info.ts:154]: here the initial-credit-limit is extracted with the '$' sign

**Review by chavanr2019 — COMMENTED** [review]
@jmthibault79 The test has passed locally. Please review and let me know if you want me to make any changes. Thank you for your time!

File: e2e/app/page/admin-user-profile-info.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ const DataTestIdAlias = {
   Username: 'username',
   ContactEmail: 'contactEmail',
   InitialCreditsUsed: 'initial-credits-used',
-  InitialCreditsDropdown: 'initial-credits-dropdown',
```
[the remark this answers is not in the document]
Comment (chavanr2019) [reply e2e/app/page/admin-user-profile-info.ts:19]: I've tried to match it with labels.

File: e2e/app/page/admin-user-profile-info.ts (hunk 1 above)
Comment (chavanr2019) [reply e2e/app/page/admin-user-profile-info.ts:141]: Ok! will update it. Thanks!

Comment (chavanr2019) [reply e2e/app/page/admin-user-profile-info.ts]: done!

**Pull request merged**